exchanges/gemini/gemini_wrapper.go

```diff
@@ -918,3 +923,32 @@ func (g *Gemini) GetHistoricCandlesExtended(_ context.Context, _ currency.Pair,
 func (g *Gemini) GetFuturesContractDetails(context.Context, asset.Item) ([]futures.Contract, error) {
 	return nil, common.ErrFunctionNotSupported
 }
+
+// UpdateOrderExecutionLimits sets exchange executions for a required asset type
+func (g *Gemini) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return fmt.Errorf("%w %v", asset.ErrNotSupported, a)
+	}
+	details, err := g.GetSymbolDetails(ctx, "all")
+	if err != nil {
+		return fmt.Errorf("cannot update exchange execution limits: %w", err)
+	}
+	resp := make([]order.MinMaxLevel, 0, len(details))
+	for i := range details {
+		if !strings.EqualFold(details[i].Status, "open") {
```

Comment:
```go
if !strings.EqualFold(details[i].Status, "open") && !strings.EqualFold(details[i].Status, "limit_only") {
			continue
		}
```

There are 14 assets that will not match but can be deployed to the books. 

example:
```
{"base_currency":"ZEC","contract_price_currency":"ETH","contract_type":"vanilla","min_order_size":"0.001","product_type":"spot","quote_currency":"ETH","quote_increment":0.00001,"status":"limit_only","symbol":"ZECETH","tick_size":0.000001,"wrap_enabled":false}
```


I always like to make sure all the stored pairs can match against whats loaded

```go
	pew, err := g.GetAvailablePairs(asset.Spot)
	if err != nil {
		t.Fatal(err)
	}

	g.SetPairs(pew, asset.Spot, true)

	for x := range pew {
		limit, err := g.GetOrderExecutionLimits(asset.Spot, pew[x])
		if err != nil {
			fmt.Println(pew[x])
			t.Fatal(err)
		}
		if limit == (order.MinMaxLevel{}) {
			t.Fatal("exchange limit should be loaded")
		}
	}
```

Reply:
Part of me wishes to only support fully actively tradable pairs, but I'll add support for such 💩 that can't trade properly